pysnmp/meta.yaml

```diff
@@ -0,0 +1,90 @@
+{% set name = "pysnmp" %}
+{% set version = "4.4.2" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "aaed6e807c26c3a38257daeb851b7a4354df011aa00ed77bd48ef3c4dd8b467b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.{{ file_ext }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  {{ hash_type }}: {{ hash_value }}
+
+build:
```

Comment:
looks like it could possibly be `noarch: python`:

http://conda-forge.readthedocs.io/en/latest/meta.html#building-noarch-packages

Reply:
Added.